pyquaticus/envs/pyquaticus.py

```diff
@@ -244,6 +245,34 @@ def reset(self):
         self.has_flag = False
         self.on_sides = True
 
+    def rotate(self):
```

Comment:
Since it would be very easy, what do you think about making this take an angle input? It could default to 180 degrees, but that might make it more reusable.